process_report/process_report.py

```diff
@@ -440,6 +440,8 @@ def merge_csv(files):
             dtype={
                 COST_FIELD: pandas.ArrowDtype(pyarrow.decimal128(12, 2)),
                 RATE_FIELD: str,
+                PI_FIELD: "string",  # Use pandas string dtype for proper string handling
```

Comment:
If you're already specifying the type for the columns here, would that make you changes in `process_report/processors/coldfront_fetch_processor.py` redundant?

Also,[ please use `pandas.StringDtype()` instead](https://github.com/CCI-MOC/invoicing/pull/198#discussion_r2110356819).